dls-controls/malcolmjs
# Feature/layout optimistic update #332

## Description

Added loading for blocks on the layout, also put stages back into the build to make it go faster. I've also updated the UglifyJS plugin for CRA to make it go in parallel and improve the build time.

I accidentally fixed #326 by refactoring how the layout engine is built (i.e. using common code).

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Start the site and view the layout loading, update a label to see it load, delete a block to see it disappear immediately.
- Check out the travis build to see the various stages.

## Agile board tracking

connect to #332 
closes #332 
connect to #326 
closes #326
